src/index.html

```diff
@@ -20,8 +20,372 @@
       href="styles/index.scss"
     />
```

Comment:
According to the checklist, styles should be linked with a `.css` extension, not `.scss`. Make sure you compile your SCSS to CSS and link the compiled stylesheet here.